pom.xml

```diff
@@ -99,12 +99,11 @@
         <h2.version>2.2.224</h2.version>
         <hibernate-orm.plugin.version>6.2.13.Final</hibernate-orm.plugin.version>
         <hibernate.c3p0.version>6.2.13.Final</hibernate.c3p0.version>
-        <infinispan.version>15.0.5.Final</infinispan.version>
+        <infinispan.version>15.0.6.Final</infinispan.version>
         <protostream.version>5.0.7.Final</protostream.version>
 
         <!--JAKARTA-->
-        <jakarta.mail.version>2.1.1</jakarta.mail.version>
-        <angus.mail.version>2.0.1</angus.mail.version>
+        <jakarta.mail.version>2.0.1</jakarta.mail.version>
```

Comment:
This is a further alignment with Quarkus: https://github.com/quarkusio/quarkus/blob/3.13.0/bom/application/pom.xml#L77